infra/src/main/scala/scaladex/infra/util/DoobieUtils.scala

```diff
@@ -147,11 +146,13 @@ object DoobieUtils {
       Meta[String].timap(fromJson[List[License]](_).get.toSet)(toJson(_))
     implicit val resolverMeta: Meta[Resolver] =
       Meta[String].timap(Resolver.from(_).get)(_.name)
-    implicit val instantMeta: doobie.Meta[Instant] = doobie.postgres.implicits.JavaTimeInstantMeta
+    implicit val instantMeta: Meta[Instant] = doobie.postgres.implicits.JavaTimeInstantMeta
 
-    implicit val projectReferenceRead: doobie.Read[Project.Reference] =
+    implicit val categoryRead: Meta[Category] = Meta[String].timap(Category.byLabel)(_.label)
```

Comment:
it's weird there is not `.get ` in `Category.byLabel`

Reply:
there is an `apply` method on `Map`